src/main/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtils.java

```diff
@@ -454,6 +454,19 @@ public static boolean doRowsExistFor(String projectID, String datasetName, Strin
         String template = "SELECT COUNT(*) FROM `%s.%s.%s` WHERE %s = %s";
```

Comment:
I'm surprised this works without having quotes around the value now that it's being used for hashes (i.e. actual Strings)

Reply:
yeah, that seems wrong -- fixed